tests/compile-fail/concurrency/libc_pthread_create_main_terminate.stderr

```diff
@@ -1,4 +1,5 @@
-warning: thread support is experimental and incomplete: weak memory effects are not emulated.
+warning: thread support is experimental: the scheduler is not preemptive, and can get stuck in spin loops.
+         (see https://github.com/rust-lang/miri/issues/1388)
```

Comment:
That is odd, I wonder where the indentation here is coming from... not a big deal though.